paddle/fluid/platform/temporary_allocator_test.cc

```diff
@@ -117,11 +117,11 @@ TEST(temporary_allocator, test_times_excess_than_required_tmp_allocation) {
   {
     PADDLE_ENFORCE_EQ(gpu_alloc.TemporaryAllocationQueueSize(), 0);
     auto tmp_allocation1 =
-        gpu_alloc.Allocate(static_cast<size_t>(100 * excess_fraction - 1));
+        gpu_alloc.Allocate(static_cast<size_t>(200 * excess_fraction - 1));
     tmp_allocation_ptr1 = tmp_allocation1->ptr();
   }
   PADDLE_ENFORCE_EQ(gpu_alloc.TemporaryAllocationQueueSize(), 1);
-  auto tmp_allocation2 = gpu_alloc.Allocate(100);
+  auto tmp_allocation2 = gpu_alloc.Allocate(200 * excess_fraction - 10);
```

Comment:
Why do you make those changes?

Reply:
Because `allocation->size()` may be larger than `requested_size` in the new design. For example, `allocation->size()` is always aligned to 256 in GPU. Therefore, `allocation->size()` would be larger than `requested_size` in most cases. In this way, this unittest would fail.